getting_started/openshift-overview.adoc

```diff
@@ -100,7 +100,7 @@ be reviewed by cluster administrators and xref:../operators/admin/olm-adding-ope
 
 * **xref:../scalability_and_performance/scaling-cluster-monitoring-operator.adoc#scaling-cluster-monitoring-operator[Scale] and xref:../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[tune] clusters**: Set cluster limits, tune nodes, scale cluster monitoring, and optimize networking, storage, and routes for your environment.
 
-* **xref:../updating/installing-update-service.adoc#installing-update-service[Understanding the OpenShift Update Service]**: Learn about installing and managing a local OpenShift Update Service for recommending {product-title} updates in restricted network environments.
+* **xref:../updating/updating-restricted-network-cluster.adoc#update-service-overview_updating-restricted-network-cluster[Understanding the OpenShift Update Service]**: Learn about installing and managing a local OpenShift Update Service for recommending {product-title} updates in restricted network environments.
```

Comment:
This is now resolved in `main` in the mainstream. Please rebase this PR and check that `getting_started/openshift-overview.adoc` is no longer listed in the **Files changed** tab.